app/core/api/serializers.py

```diff
@@ -38,6 +39,22 @@ class Meta:
         )
 
 
+class UserPermissionSerializer(serializers.ModelSerializer):
+    """Used to retrieve user permissions"""
+
+    class Meta:
+        model = UserPermission
+        fields = (
+            "uuid",
+            "created_at",
+            "updated_at",
+            "user",
+            "permission_type",
+            "project",
+            "practice_area",
+        )
```

Comment:
This is missing the read_only fields that the other models in the same file have. Actually, I'm not sure if they're settable from the API if you didn't declare them `read_only`.